independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/app/StartupAction.java

```diff
@@ -26,4 +26,9 @@
      */
     RunningQuarkusApplication runMainClass(String... args) throws Exception;
 
+    /**
+     * Runs the application by running the main method of the main class, and runs it to execution
```

Comment:
Did you mean completion?
```suggestion
     * Runs the application by running the main method of the main class, and runs it to completion
```